.github/tests/javascript/graphql-auth/run.sh

```diff
@@ -9,4 +9,4 @@ sleep 15
 
 curl --fail 'http://localhost:4000/' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Connection: keep-alive' -H 'DNT: 1' -H 'Origin: http://localhost:4000' --data-binary '{"query":"query {\n  feed {\n    id\n    content\n    author {\n      id\n      name\n      email\n    }\n  }\n}"}' --compressed
 
-kill "$pid"
+kill "$pid" || true
```

Comment:
Why should this ever not work?

Reply:
No idea but it failed on my local once and the existing code has a catch-all to clean up. This is just a precautionary measure.

Reply:
Please add an issue to investigate and understand this (or replace that code).

Reply:
Removed in https://github.com/prisma/prisma-examples/pull/2068